src/resources/Account/index.ts

```diff
@@ -48,6 +52,32 @@ export interface IAccountWithdrawConsentOpts {
   reason: 'holder_withdrew_consent' | null;
 };
 
+export interface IAccount {
+  id: string;
+  holder_id: string;
+  status: TAccountStatuses;
+  type: TAccountTypes | null;
+  ach?: IAccountACH | null;
+  liability?: IAccountLiability | null;
+  clearing?: null;
```

Comment:
Think I remembered an eng chat question about this, but just making sure its typed intentionally that this field will always be null if its included?

Reply:
@dwilliams27 that field only shows up when consent is withdrawn, which means it will always be null.  I left it in there for now but will remove if it gets taken out of the API response